src/figuresClasses.ts

```diff
@@ -1,11 +1,91 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+}
+
+export class Triangle implements Figure {
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  shape: string = 'triangle';
+
+  color: string;
 
-export class Triangle implements Figure {}
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Length of some value is less than 0');
```

Comment:
The error message 'Length of some value is less than 0' is misleading. It should be 'Length of some value is less than or equal to 0' to accurately reflect the condition being checked.